### PR TITLE
Summary one size fits all

### PR DESCRIPTION
**Pull request by mattkrick**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,32 +0,0 @@
-
```
Comment (mattkrick) [inline .env.example]: whoops, i'll add this back in

File: src/server/email/sendEmail.js [hunk 2]
```diff
@@ -37,12 +37,18 @@ export default async function sendEmailPromise(to, template, props) {
   if (!emailFactory) {
     throw new Error(`Email template for ${template} does not exist!`);
   }
-  const renderedEmail = emailFactory(props);
-  const emailWithAttachments = createEmbeddedImages(renderedEmail);
+  if (typeof to !== 'string') {
```
Comment (mattkrick) [inline src/server/email/sendEmail.js]: should check for '' as well

File: src/server/emailSSR.js [hunk 3]
```diff
@@ -1,28 +1,33 @@
-import React from 'react';
```
Comment (mattkrick) [inline src/server/emailSSR.js]: should we nuke the file contents?

File: src/server/graphql/models/Meeting/meetingMutation.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+      requireSUOrTeamMember(authToken, teamId);
+      if (!summarySentAt) {
+        // send the summary email
+        const userIds = invitees.map((doc) => doc.id.substr(0, doc.id.indexOf('::')));
+        const emails = await r.table('User')
+          .getAll(r.args(userIds))
+          .map((user) => user('email'));
+        const emailString = emails.join(', ');
+        if (!emailString) {
+          console.log('NULL STR', meeting, userIds, emails);
```
Comment (mattkrick) [inline src/server/graphql/models/Meeting/meetingMutation.js]: TODO: still need to figure out why I sometimes get a meeting with `invitees: []`

File: src/server/graphql/models/Meeting/meetingSchema.js [hunk 5]
```diff
@@ -166,7 +185,10 @@ export const Meeting = new GraphQLObjectType({
       description: 'The timestamp the meeting officially ended'
     },
     invitees: {
-      type: new GraphQLList(MeetingInvitee)
+      type: new GraphQLList(MeetingInvitee),
+      // async resolve({invitees, actions, projects}) {
```
Comment (mattkrick) [inline src/server/graphql/models/Meeting/meetingSchema.js]: will remove

File: .env.example [hunk 6]
```diff
@@ -1,32 +0,0 @@
...
-# global container options
```
Comment (jordanh) [inline .env.example]: I don't think we meant to remove this file, right?

File: package.json [hunk 7]
```diff
@@ -3,7 +3,7 @@
   "description": "An open-source app for building smarter, more agile teams.",
   "author": "Parabol Inc. <love@parabol.com> (http://github.com/ParabolInc)",
   "license": "AGPL-3.0",
-  "version": "0.8.1",
+  "version": "0.9.2",
```
Comment (jordanh) [inline package.json]: Let's bump to 0.9.3

File: src/server/graphql/models/Team/teamMutation.js [hunk 8]
```diff
@@ -273,11 +278,21 @@ export default {
...
+          facilitatorPhaseItem: null,
+          meetingPhaseItem: null,
+        });
+
       // reset the meeting
       await r.table('Team').get(teamId)
         .update({
-          facilitatorPhase: 'lobby',
-          meetingPhase: 'lobby',
+          facilitatorPhase: LOBBY,
```
Comment (jordanh) [inline src/server/graphql/models/Team/teamMutation.js]: Good catch!

File: src/universal/modules/email/containers/SummaryEmail/SummaryEmail.js [hunk 9]
```diff
@@ -1,144 +0,0 @@
-import React, {PropTypes} from 'react';
```
Comment (jordanh) [inline src/universal/modules/email/containers/SummaryEmail/SummaryEmail.js]: I had it on my PR checklist to make sure this file was removed. Nice work! ✔️

File: src/universal/containers/Action/ActionContainer.js [hunk 10]
```diff
@@ -1,39 +1,48 @@
...
     location: PropTypes.shape({
       pathname: PropTypes.string.isRequired
     }).isRequired
   };
 
   componentWillMount() {
-    const {location: {pathname: nextPage}} = this.props;
-    updateAnalyticsPage('', nextPage);
+    const {dispatch, location: {pathname: nextPage}} = this.props;
+    updateAnalyticsPage(dispatch, '', nextPage);
```
Comment (mattkrick) [inline src/universal/containers/Action/ActionContainer.js]: @jordanh i still get a bunch of warnings in my console due to segment (window.analytics doesn't exist). Should we make segment a production-only thing? if not, could we check for the presence of window.analytics in ActionContainer?

File: src/universal/modules/email/components/QuickStats/QuickStats.js [hunk 11]
```diff
@@ -0,0 +1,117 @@
...
+      <div style={headingStyles}>
+        Quick Stats
+      </div>
+      <table width="100%">
+        <tbody>
+          <tr>
+            <td style={cellStyles}>
+              <div style={statStyles}>
+                <div style={statValue}>{agendaItems}</div>
+                <div style={statLabel}>Agenda Item{agendaItems !== 1 && 's'}</div>
```
Comment (mattkrick) [inline src/universal/modules/email/components/QuickStats/QuickStats.js]: will convert this to use the `plural` utility function

File: src/universal/modules/email/components/QuickStats/QuickStats.js [hunk 12]
```diff
@@ -0,0 +1,117 @@
...
+
+QuickStats.propTypes = {
+  agendaItems: PropTypes.number,
+  newProjects: PropTypes.number,
+  newActions: PropTypes.number,
+  teamMembers: PropTypes.number,
+  teamMembersPresent: PropTypes.number
+};
+
+QuickStats.defaultProps = {
```
Comment (mattkrick) [inline src/universal/modules/email/components/QuickStats/QuickStats.js]: will remove

File: src/universal/modules/email/helpers/sampleTeamSummary.js [hunk 13]
```diff
@@ -1,11 +1,14 @@
 const avatarJordan = '/static/images/avatars/jh-linkedin-avatar.jpg';
```
Comment (mattkrick) [inline src/universal/modules/email/helpers/sampleTeamSummary.js]: @ackernaut can delete?

Comment (ackernaut) [inline src/universal/modules/email/helpers/sampleTeamSummary.js]: Go for it!

**Pull request merged**